src/components/Builder/UpsellItems.tsx

```diff
@@ -113,9 +115,9 @@ const UpsellItems = ({
         >
           {categoriesItems.map(({ category, item }) => {
             const optionGroups = item ? item.option_groups || [] : []
-            const isSize = !!(optionGroups.length && optionGroups[0].is_size)
+            const hasOptions = !!optionGroups.length
             return itemFilter(item) ? (
-              isSize ? (
+              hasOptions ? (
```

Comment:
@franciscoxrra Is this right? I believe that only items with an `is_size` modifier group (such as Fountain Drinks) should be expanded?